Dockerfile

```diff
@@ -43,4 +43,4 @@ RUN echo "source /pwndbg/gdbinit.py" >> ~/.gdbinit.py && \
 
 ADD . /pwndbg/
 
-RUN git submodule update --init --recursive
+RUN git submodule update --init --recursive
```

Comment:
Can you add back the newline? A lot of editors enforce this, so it's going to get changed again the next time someone with that editor config touches this file.

Reply:
will do